app/assets/stylesheets/layouts/header.scss

```diff
@@ -2,6 +2,9 @@
   height: 60px;
   background-color: $theme-primary;
   display: flex;
+  position: sticky;
+  z-index: 9999;
```

Comment:
let's have these arbitrary z-index values as variables. Also you don't need to have it so high

Reply:
going to put a tech debt on this so i can merge.  need to get branches cleared out so I can get my own merges in on the linting.